setup.cfg

```diff
@@ -49,17 +49,19 @@ include =
 
 [options.extras_require]
 opt =
-    opencv
-    openh264
+    opencv-contrib-python-headless
     pytransform3d
     noisyopt
+    scikit-learn
+    pyyaml
 test =
     pytest
     pytest-xdist
-    pyyaml
     flake8
     pylint
+    scikit-learn
     richdem
+    gdal
```

Comment:
cf my global remark, It would be better to remove gdal if possible, in my opinon... in the future ?